### DISCUSSION TITLE
post_233845

### DISCUSSION THREAD

**Discussion by cincuranet** [cincuranet/blog-comments#1349] [Comments]
imported 

https://www.tabsoverspaces.com/id/233845

**Comment by cincuranet** [comment]
Imported comment written by _Zachary Snow_ on _2020-11-20T14:26:53_:

<p>I'll be attending and looking forward to your presentation.</p>

**Reply by itsSketchyHere** [reply]
Is it now available on youtube or any other place ?

**Reply by cincuranet** [reply]
I'm not aware of any place where it would be available.

**Comment by itsSketchyHere** [comment]
hey when I search on youtube - "NDC Tasks and async/await from the ground up"  I cannpt find your video there. Also this [link](https://www.dotnet-developer-conference.de/programm%202020/#/talk/tasks-and-async-await-from-the-ground-up) isn't working., it says File Not Found.

**Reply by cincuranet** [reply]
Yeah, it's been 3 years since this took place. The website was surely modified.
